cli.js

```diff
@@ -1047,6 +1047,8 @@ program
     .option('-h, --host <host>','hostname alias to register')
     .option('-j, --json', 'Optional, formats the output in json')
     .option('-u, --unique', 'Optional, define alias as unique, false by default')
+    .option('-l, --request-letsencrypt-certificate <requestLetsncrypt>', 'Optional, ' +
```

Comment:
Boolean flags should be passed as name only arguments to overwrite the default value (false in this case). So, instead of:
```
sfcc-ci sandbox:alias:create --request-letsencrypt-certificate true
```
it should be
```
sfcc-ci sandbox:alias:create --request-letsencrypt-certificate
```

Reply:
fixed